src/securechip/securechip.c

```diff
@@ -78,7 +78,7 @@ typedef union {
 
 #pragma GCC diagnostic pop
 
-static securechip_interface_functions_t* _interface_functions = NULL;
+static const securechip_interface_functions_t* _interface_functions = NULL;
```

Comment:
there's also common_main.c: https://github.com/benma/bitbox02-firmware/blob/5715aeb2/src/common_main.c#L42-L53
not sure why - the static structs in common_main seems unnecessary but maybe I'm missing somehitng. slightly confused.

Reply:
They provide the actual functions to be used by memory/securechip in the final firmware (not unit test).

I made them const as well :+1: